tools/cldr-code/src/main/java/org/unicode/cldr/util/XMLSource.java

```diff
@@ -590,6 +590,7 @@ public String getSourceLocaleIdExtended(
                 list.add(
                         new LocaleInheritanceInfo(
                                 locale, path, LocaleInheritanceInfo.Reason.value));
+                // Since we’re not resolving, there’s no way to look for a Bailey value here.
```

Comment:
We should throw an exception if we try for a bailey value with a non-resolving source.

Reply:
Good point. and i'll add an exception up in `getPathsWhereFound()`, certainly.  This code, however is two levels deeper, in the XMLSource override.

Reply:
ok. Are you going to update the test as part of this PR?

Reply:
Yes, I'll probably work on that Monday.

Reply:
Test found some bugs !

Reply:
throws exception, passes tests